tests/integration/goldens/redis/google/cloud/redis_v1/services/cloud_redis/transports/rest_asyncio.py

```diff
@@ -250,29 +254,149 @@ class _GetInstance(_BaseCloudRedisRestTransport._BaseGetInstance, AsyncCloudRedi
         def __hash__(self):
             return hash("AsyncCloudRedisRestTransport.GetInstance")
 
+        @staticmethod
+        async def _get_response(
+            host,
+            metadata,
+            query_params,
+            session,
+            timeout,
+            transcoded_request,
+            body=None):
+
```

Comment:
Add type hints and docstring for this method

Reply:
This is a protected helper which pre-existed and is not a part of this PR's changed. I see that it's now also leveraged for async rest. Regardless, we can address this as a follow up. Let me know if that sounds good!